src/pages/components/draggable.mdx

```diff
@@ -0,0 +1,233 @@
+  <DraggablePlaceholder />
+</CodeExample>
+
+### Size
+
+Can be default or compact in size. This changes the vertical padding.
+
+import DraggableSize from '../../examples/draggable/DraggableSize.tsx';
+import DraggableSizeCode from '!!raw-loader!../../examples/draggable/DraggableSize.tsx';
+
+<CodeExample code={DraggableSizeCode}>
+  <DraggableSize />
+</CodeExample>
+
+## Draggable list
+
+### Default
+
+Use the list component to organize `Draggable` items into a vertical list.
+
+import DraggableListDefault from '../../examples/draggable/DraggableListDefault.tsx';
+import DraggableListDefaultCode from '!!raw-loader!../../examples/draggable/DraggableListDefault.tsx';
+
+<CodeExample code={DraggableListDefaultCode}>
+  <DraggableListDefault />
+</CodeExample>
+
+### Horizontal
+
+Use `isHorizontal` to display items in a row.
+
+import DraggableListHorizontal from '../../examples/draggable/DraggableListHorizontal.tsx';
+import DraggableListHorizontalCode from '!!raw-loader!../../examples/draggable/DraggableListHorizontal.tsx';
+
+<CodeExample code={DraggableListHorizontalCode}>
+  <DraggableListHorizontal />
+</CodeExample>
+
+### Indicator
+
+Use indicators to communicate where the `Draggable` will land between other items.
+
+import DraggableListIndicator from '../../examples/draggable/DraggableListIndicator.tsx';
+import DraggableListIndicatorCode from '!!raw-loader!../../examples/draggable/DraggableListIndicator.tsx';
+
+<CodeExample code={DraggableListIndicatorCode}>
+  <DraggableListIndicator />
+</CodeExample>
+
+## Dropzone
+
+### Default
+
+The Dropzone allows users to designate an area as a drop target. This example uses [dnd-kit](https://github.com/clauderic/dnd-kit).
+
+import DropzoneDefault from '../../examples/draggable/DropzoneDefault.tsx';
+import DropzoneDefaultCode from '!!raw-loader!../../examples/draggable/DropzoneDefault.tsx';
+
+<CodeExample code={DropzoneDefaultCode}>
+  <DropzoneDefault />
+</CodeExample>
+
+### Danger
+
+Danger styling communicates that items dropped will be removed or deleted. This example uses [dnd-kit](https://github.com/clauderic/dnd-kit).
+
+import DropzoneDanger from '../../examples/draggable/DropzoneDanger.tsx';
+import DropzoneDangerCode from '!!raw-loader!../../examples/draggable/DropzoneDanger.tsx';
+
+<CodeExample code={DropzoneDangerCode}>
+  <DropzoneDanger />
+</CodeExample>
+
+## Configuration
+
+<Configuration reactPackage={props.data.mdx.package} components={props.data.mdx.components} />
+
+## API
+
+### Draggable
+
+<Component components={props.data.mdx.components} componentName="draggable" />
+
+<PropSheet components={props.data.mdx.components} componentName="draggable" />
+
+#### Draggable.Content
+
+<Component components={props.data.mdx.subcomponents} componentName="draggable.content" />
+
+#### Draggable.Grip
+
+<Component components={props.data.mdx.subcomponents} componentName="draggable.grip" />
+
+Provides visual feedback that a `Draggable` can be grabbed and moved.
```

Comment:
```suggestion
Provides visual indication that a `Draggable` can be grabbed and moved.
```
I think of "feedback" as a response triggered by a user's action.

Reply:
ah fair point, agreed!